website/content/prototyping/_index.markdown

```diff
@@ -1,195 +1,17 @@
 ---
 layout: default
-title: Non-Go Prototyping
+title: Package development
 pre: "<b>5. </b>"
 weight: 5
-aliases:
-  - /prototyping.html
+chapter: true
 ---
 
-To realize the full benefits of gokrazy, you need to use only
-software written in Go. If there is no Go software for what you want
-to do, creating that piece of software can pose a seemingly
-unsurmountable hurdle. To make some quick progress and figure out if
-your idea can be implemented, it might make sense to temporarily use
-existing software before starting your own implementation.
+# Package development
 
-This article shows a couple of techniques for getting non-Go
-software to work on gokrazy, in increasing order of complexity.
+A gokrazy package is simply the `main` package of a Go software.
```

Comment:
That’s not entirely correct, actually. A gokrazy package is a Go package.

Even if the Go package declares a named library instead of a `main` package, that’s still a valid gokrazy package, and can be useful to install. https://github.com/gokrazy/podman is an example for that: it declares `package podman` in `podman.go`, so it won’t result in an extra binary (no `/user/podman`). But, because it also contains a `_gokrazy/extrafiles.tar`, those files will be included in the root file system.